lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1513,8 +1513,9 @@ void FIRStmtParser::emitInvalidate(Value val, Flow flow) {
   // to only the leaf sources.
   TypeSwitch<FIRRTLType>(tpe)
       .Case<BundleType>([&](auto tpe) {
-        for (auto element : tpe.getElements()) {
-          auto subfield = builder.create<SubfieldOp>(val, element.name);
+        for (size_t index = 0, size = tpe.getNumElements(); index < size;
+             ++index) {
+          auto subfield = builder.create<SubfieldOp>(val, index);
```

Comment:
Nit: can save a line:
```suggestion
        for (size_t i = 0, e = tpe.getNumElements(); i != e; ++i) {
          auto subfield = builder.create<SubfieldOp>(val, i);
```